controllers/auth.go

```diff
@@ -725,6 +725,10 @@ func (c *ApiController) Login() {
 						Properties:        properties,
 					}
 
+					if user.Phone != "" || user.Email != "" {
```

Comment:
Don't let everyone reset the password. Let admin control it in UI of user edit page